packer/http/autoyast.xml.tpl

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+
+  <bootloader>
+    <global>
+      <append>cgroup_enable=memory swapaccount=1</append>
+    </global>
+  </bootloader>
+
+  <firewall>
+    <enable_firewall config:type="boolean">false</enable_firewall>
+    <start_firewall config:type="boolean">false</start_firewall>
+  </firewall>
+
+  <keyboard>
+    <keymap>english-us</keymap>
+  </keyboard>
+
+  <language>
+    <language>en_US</language>
+    <languages>en_US</languages>
+  </language>
+
+  <networking>
+    <dhcp_options>
+      <dhclient_client_id/>
+      <dhclient_hostname_option>AUTO</dhclient_hostname_option>
+    </dhcp_options>
```

Comment:
I know you didn't change this, but I wonder if we could drop the whole `<dhcp_options>` block?

Not worth a rebuild to test, though